skip_toc_check.txt

```diff
@@ -35,6 +35,7 @@ docs/contribute/page-templates.md
 docs/contribute/review-issues.md
 docs/contribute/stage-documentation-changes.md
 docs/contribute/style-guide.md
+docs/contribute/includes.md
 docs/contribute/write-new-topic.md
```

Comment:
Why would we skip the TOC check for includes.md?

Reply:
I don't know, I was just following convention from the other pages in this section. @chenopis might have a better idea there??